src/site/content/en/animations/animations-guide/index.md

```diff
@@ -105,52 +71,51 @@ you could use `transform` and all of the work could be done in the composite ste
   }
 }
 ```
+{% endCompare %}
 
-[See this example on Glitch](https://glitch.com/~animation-with-transform).
+## Resize an element {: #resize }
 
-Test this example in Chrome and Firefox DevTools to see the difference that the change makes.
+To resize an element, use the `scale` keyword value of the
+[`transform`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform) property.
 
-## Finding paint problems
+## Change an element's visibility {: #visibility }
 
-DevTools can help you to identify which parts of your page are being repainted.
-In Chrome DevTools, open the Rendering Tab and select Paint Flashing.
-In Firefox DevTools add a Toolbox button for [Paint Flashing](https://developer.mozilla.org/en-US/docs/Tools/Paint_Flashing_Tool).
-These tools will highlight areas of the page that will be repainted.
+To show or hide an element, use [`opacity`](https://developer.mozilla.org/en-US/docs/Web/CSS/opacity).
 
-<figure class="w-figure">
-  <img class="w-screenshot" src="./paint-flashing.jpg" alt="A UI element highlighted in green to demonstrate it will be repainted">
-</figure>
+<!-- TODO(kaycebasques): Add basic examples -->
 
-If you see the whole screen flashing,
-or areas that you don't think should change highlighted then you can do some investigation.
+## Avoid properties that trigger layout or paint {: #triggers }
 
-When it comes to painting, some things are more expensive than others.
-For example, anything that involves a blur (like a shadow, for example) is going to take longer to paint than drawing a red box.
-In terms of CSS, however, this isn't always obvious:
-`background: red;` and `box-shadow: 0, 4px, 4px, rgba(0,0,0,0.5);`
-don't necessarily look like they have vastly different performance characteristics, but they do.
+Before using any CSS property for animation (other than `transform` and `opacity`), go to
+[CSS Triggers](https://csstriggers.com/) to determine the property's impact on the
+[rendering pipeline](/animations-overview/#pipeline). Avoid any property
+that triggers layout or paint unless absolutely necessary. 
 
-If you need to dig into whether a particular property is causing performance issues due to painting,
-the [paint profiler](https://developers.google.com/web/tools/chrome-devtools/evaluate-performance/reference#paint-profiler)
-in Chrome DevTools can help.
+<figure class="w-figure">
+  <img class="w-screenshot" src="./css-triggers.jpg" alt="The top property detailed on CSS Triggers">
+</figure>
+
+{% Aside 'warning' %}
+  If you must use a property
+  that triggers layout or paint, it is unlikely that you will be able to make the animation
+  smooth and high-performance.
+{% endAside %}
 
-## Forcing layer creation
+## Force layer creation {: #force }
```

Comment:
Just rephrasing this so that it's consistent with the other section heading changes I made